Android.gitignore

```diff
@@ -63,3 +57,6 @@ fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
 fastlane/readme.md
+
+# Mac temp image file
+.DS_Store
```

Comment:
This is an OS-specific rule, and is already covered in `Global/macOS.gitignore`. Please remove it from this template.